src/lambdas/api/app.js

```diff
@@ -129,17 +129,10 @@ app.get('/collections/:collectionId/items', async (req, res, next) => {
 })
 
 app.post('/collections/:collectionId/items', async (req, res, next) => {
-  try {
-    res.json(await api.searchItems(
-      req.params.collectionId,
-      req.body,
-      satlib.es,
-      req.endpoint,
-      'POST'
-    ))
-  } catch (error) {
-    next(error)
-  }
+  if (!process.env['ENABLE_TRANSACTIONS_EXTENSION']) next(createError(404))
```

Comment:
Rather than just checking for the presence of this environment variable, do we want to be making sure it's equal to something? Setting it to `false`, for example, would have unexpected results.

Reply:
lol, I just copied what was already there. Good point though, I'll fix all of them.

Reply:
and turning off the txn api would have never worked, since this was populating the value of the enabled env var as the string "false"